pkg/operator/certrotation/cabundle.go

```diff
@@ -88,14 +89,20 @@ func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingC
 	}
 
 	if modified {
```

Comment:
This code is a bit hard to read. I would like to make it more explicit. Given that we would export the event reporting methods. We could do:

```
	if creationRequired {
		actualCABundleConfigMap, err := c.Client.ConfigMaps(c.Namespace).Create(ctx, caBundleConfigMap, metav1.CreateOptions{})
		resourcehelpers.ReportCreateEvent(c.EventRecorder, actualCABundleConfigMap, err)
		if err != nil {
			return nil, err
		}
		klog.V(2).Infof("Created ca-bundle.crt configmap %s/%s with:\n%s", certs.CertificateBundleToString(updatedCerts), caBundleConfigMap.Namespace, caBundleConfigMap.Name)
		caBundleConfigMap = actualCABundleConfigMap
	} else if updateRequired {
		actualCABundleConfigMap, err := c.Client.ConfigMaps(c.Namespace).Update(ctx, caBundleConfigMap, metav1.UpdateOptions{})
		resourcehelpers.ReportUpdateEvent(c.EventRecorder, actualCABundleConfigMap, err)
		if err != nil {
			return nil, err
		}
		klog.V(2).Infof("Updated ca-bundle.crt configmap %s/%s with:\n%s", certs.CertificateBundleToString(updatedCerts), caBundleConfigMap.Namespace, caBundleConfigMap.Name)
		caBundleConfigMap = actualCABundleConfigMap
	}
```

WDYT ?